_data/analysis-files.yml

```diff
@@ -0,0 +1,36 @@
+- file: continent-estimated.csv
```

Comment:
I propose we add a github workflow to check if there is a description for each file in assets/csv, so that we won't have this discrepancy between files that do have an alternative name and those that don't

Reply:
This will be helpful!